_episodes/03-lists.md

```diff
@@ -38,7 +38,10 @@ odds are: [1, 3, 5, 7]
 ~~~
 {: .output}
 
-We select individual elements from lists by indexing them:
+We select individual elements from lists by indexing them.
+Indexing allows us to access individual elements inside a container (such as list or string).
+Previously, we used indexing to count elements starting from the beginning, but we can also count from the end of the container using negative indices.
+We will discuss this further when we come to the topic of "slicing":
```

Comment:
I suggest moving (and slightly modifying) this discussion to right after the code block. I don't think it is worth to explain negative indexing before they see it for the first time. I think this is one of those moments... "Look what I can do in Python".... "And here is why". Perhaps, it's worth even splitting the code block into `odd[0]`, `odd[3]` and `odd[-1]`. But

Reply:
Hi @maxim-belkin. I agree with that - I've rewritten the Markdown along those lines. Let me know if you want further changes (I think that the current phrasing might be a bit awkward).